config/files/usr/share/ublue-os/just/60-custom.just

```diff
@@ -2,14 +2,85 @@
 
 enable-cups:
-    firewall-cmd --add-service=cups --permanent 
-    firewall-cmd --reload 
-    systemctl unmask cups
-    systemctl enable cups
-    systemctl start cups
+    firewall-cmd --add-service=cups --permanent &&\
+    firewall-cmd --reload &&\
+    systemctl unmask cups &&\
+    systemctl enable --now cups &&\
+    echo "CUPS enabled."
+
+disable-cups:
+    firewall-cmd --remove-service=cups --permanent &&\
+    firewall-cmd --reload &&\
+    systemctl stop cups &&\
+    systemctl disable cups &&\
+    systemctl mask cups &&\
+    echo "CUPS disabled."
+
+userns-on
+    # test if userns are disabled
+    if grep -q "kernel.unprivileged_userns_clone = 0" "/usr/etc/sysctl.d/hardening.conf"; then
+
+        # read the container image reference and change the string
+        userns_version=$(rpm-ostree status --json | jq -r '.deployments[0]."container-image-reference"' | sed 's/hardened/userns-&/')
+        echo "Rebasing to the user namespaces image..."
+        rpm-ostree rebase $userns_version
+        echo "Reboot for the changes to apply."
+
+    # test if userns are enabled
+    elif grep -q "kernel.unprivileged_userns_clone = 1" "/usr/etc/sysctl.d/hardening.conf"; then
+        echo "User namespaces are already enabled."
+
+    else
+        echo "The check has failed, please report that error."
+        rpm-ostree status --json | jq -r '.deployments[0]."container-image-reference"'
+        xdg-open "https://github.com/secureblue/secureblue/issues/new"
+    fi
+
+userns-off
+    # test it userns are disabled
+    if grep -q "kernel.unprivileged_userns_clone = 0" "/usr/etc/sysctl.d/hardening.conf"; then
+        echo "User namespaces are already disabled."
+
+    # test if userns are enabled
+    elif grep -q "kernel.unprivileged_userns_clone = 1" "/usr/etc/sysctl.d/hardening.conf"; then
+        nouserns_version=$(rpm-ostree status --json | jq -r '.deployments[0]."container-image-reference"' | sed 's/-userns//')
+        echo "Rebasing to the image without user namespaces..."
+        rpm-ostree rebase $nouserns_version
+        echo "Reboot for the changes to apply."
+
+    else
+        echo "The check has failed, please report that error."
+        rpm-ostree status --json | jq -r '.deployments[0]."container-image-reference"'
+        xdg-open "https://github.com/secureblue/secureblue/issues/new"
+    fi
+
+bluetooth-on
+    systemctl unmask bluetooth.service &&\
+    systemctl enable --now bluetooth.service &&\
+    echo "Bluetooth enabled."
+
+bluetooth-off
+    systemctl stop bluetooth.service &&\
+    systmctl disable bluetooth.service &&\
+    systemctl mask bluetooth.service &&\
+    echo "Bluetooth disabled."
+
+flathub-all
+    flatpak remote-modify --subset=all flathub &&\
+    echo "The Flathub Repository now shows all available Apps."
+
+flathub-verified
+    flatpak remote-modify --subset=verified flathub &&\
+    echo "The Flathub Repository now shows only verified Apps."
+
```

Comment:
fedora is already generic, this is unnecessary. please remove it.

Reply:
this is the manual opt-in when users choose to use only verified apps again, e.g. when their needed apps got verified

Reply:
This comment was on the hostnamectl just command. But on the flathub commands, they are also not needed since the user can just add a second flathub remote.